common/djangoapps/third_party_auth/admin.py

```diff
@@ -2,12 +2,21 @@
 """
 Admin site configuration for third party authentication
 """
+from django import forms
 
 from django.contrib import admin
 
 from config_models.admin import ConfigurationModelAdmin, KeyedConfigurationModelAdmin
-from .models import OAuth2ProviderConfig, SAMLProviderConfig, SAMLConfiguration, SAMLProviderData, LTIProviderConfig
+from .models import (
+    OAuth2ProviderConfig,
+    SAMLProviderConfig,
+    SAMLConfiguration,
+    SAMLProviderData,
+    LTIProviderConfig,
+    ApiPermissions
```

Comment:
Can we rename this to `ProviderApiPermissions` so it is it clearly scoped in its name?